ekm/genesis_adapter.go

```diff
@@ -1,6 +1,7 @@
 package ekm
 
 import (
+	genesisphase0 "github.com/AKorpusenko/genesis-go-eth2-client/spec/phase0"
```

Comment:
did `go-eth2-client/spec/phase0` change?
also if we're using forked code, lets at least use "replace" directive, and preferably lets put it in a repo under ssv.

Reply:
if this is the reason for the file it should be removed once pec version is upgraded